### DISCUSSION TITLE
access 代替api填写求助

### DISCUSSION THREAD

**Discussion by justincnn** [linweiyuan/go-chatgpt-api#211] [General]
请问下，使用token代替api，下面填写是否正确：
- OPENAIKEY=sk-xxxxx(随便填写)
- OPENAI_BASE_URL=https://自己vps地址/imitate/v1/chat/completions

如果这样的话，那access token填写在什么地方？

**Comment by linweiyuan** [comment]
OPENAPIKEY 填 access token

**Comment by justincnn** [comment]
> OPENAPIKEY 填 access token

可以了，感谢大神
